tests/spicy/tools/spicy-driver-properties.spicy

```diff
@@ -0,0 +1,43 @@
+# @TEST-EXEC: spicyc -j -o x.hlto %INPUT
+# @TEST-EXEC: printf abc | spicy-driver -p 80/tcp x.hlto >>output
+# @TEST-EXEC: printf abc | spicy-driver -p 100/tcp%orig x.hlto >>output
+# @TEST-EXEC: printf abc | spicy-driver -p 100/tcp%resp x.hlto >>output
+# @TEST-EXEC: printf abc | spicy-driver -p application/http x.hlto >>output
+# @TEST-EXEC: btest-diff output
+#
+# Select parsers through %port or %mime-type properties.
+
+module Test;
+
+public type FooPort = unit {
+    %port = 80/tcp;
+
+    x: bytes &eod;
+    on %done { print "Port", self; }
+};
+
+public type FooPortOrig = unit {
+    %port = 100/tcp &originator;
+
+    x: bytes &eod;
+    on %done { print "Port (orig)", self; }
+};
+
+public type FooPortResp = unit {
+    %port = 100/tcp &responder;
+
+    x: bytes &eod;
+    on %done { print "Port (resp)", self; }
+};
+
+public type FooMIME = unit {
+    %mime-type = "application/http";
+
+    x: bytes &eod;
+    on %done { print "MIME", self; }
+};
```

Comment:
Can we also add one with a wildcard like e.g., `application/*`?

Reply:
we don't do that matching, so it wouldn't have any noticeable effect